src/avkit.cs

```diff
@@ -208,6 +134,14 @@ interface AVPlayerViewController {
 		[NoiOS, TV (11,2), NoMac, NoWatch]
 		[Export ("appliesPreferredDisplayCriteriaAutomatically")]
 		bool AppliesPreferredDisplayCriteriaAutomatically { get; set; }
+		
+		[iOS (9, 0), TV (13,0), NoWatch]
```

Comment:
It was probably private before iOS 13, a quick simulator test to ensure it's actually available on older versions would help.

Reply:
I just checked, it's fine : 
![BCE57F5F-FB71-4C52-BB87-43B4C612D964](https://user-images.githubusercontent.com/41507018/62904797-09ca1d80-bd68-11e9-952e-f0f08736f276.png)